app/plugins/ee-request-transformer.md

```diff
@@ -0,0 +1,34 @@
+---
+id: page-plugin
+title: Plugins - EE request Transformer
+header_title: EE request Transformer
+header_icon: /assets/images/icons/plugins/EE-request-transformer.png
+breadcrumbs:
+  Plugins: /plugins
+nav:
+  - label: Getting Started
+    items:
+      - label: Requesting Access
+---
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus aliquid ea assumenda ipsa iure officia obcaecati quod, cupiditate exercitationem mollitia animi veritatis eum deleniti, et libero quos cumque consequuntur dolore.
```

Comment:
The Request Transformer plugin for Kong Enterprise Edition builds on the Community Edition version of this plugin with enhanced capabilities to match portions of incoming requests using regular expressions, save those matched strings into variables, and substitute those strings into transformed requests via flexible templates.